pkg/udf/rpc/errors.go

```diff
@@ -44,3 +44,8 @@ func (e ApplyUDFErr) IsInternalErr() bool {
 func (e ApplyUDFErr) Error() string {
 	return fmt.Sprint(e.Message)
 }
+
+// Is checks if the error is of the same type
+func (e ApplyUDFErr) Is(target error) bool {
```

Comment:
Where is the function used?

Reply:
i think it is coming from "session" code.

Reply:
https://github.com/numaproj/numaflow/blob/main/pkg/reduce/pnf/processandforward.go#L136

Reply:
looks like it hit the never in `session` code :)